[2024-08-24T18:36:58.605+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-24T18:36:58.632+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:36:56.685766+00:00 [queued]>
[2024-08-24T18:36:58.645+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:36:56.685766+00:00 [queued]>
[2024-08-24T18:36:58.646+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-24T18:36:58.666+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): create_output_directory> on 2024-08-24 18:36:56.685766+00:00
[2024-08-24T18:36:58.674+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=812) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-24T18:36:58.677+0000] {standard_task_runner.py:72} INFO - Started process 815 to run task
[2024-08-24T18:36:58.680+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'process_employee', 'create_output_directory', 'manual__2024-08-24T18:36:56.685766+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/employee_DAG.py', '--cfg-path', '/tmp/tmpw40gi4a0']
[2024-08-24T18:36:58.681+0000] {standard_task_runner.py:105} INFO - Job 77: Subtask create_output_directory
[2024-08-24T18:36:58.755+0000] {task_command.py:467} INFO - Running <TaskInstance: process_employee.create_output_directory manual__2024-08-24T18:36:56.685766+00:00 [running]> on host 67412decceee
[2024-08-24T18:36:58.871+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_employee' AIRFLOW_CTX_TASK_ID='create_output_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T18:36:56.685766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-24T18:36:56.685766+00:00'
[2024-08-24T18:36:58.873+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-24T18:36:58.876+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-24T18:36:58.889+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-24T18:36:58.889+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_employee, task_id=create_output_directory, run_id=manual__2024-08-24T18:36:56.685766+00:00, execution_date=20240824T183656, start_date=20240824T183658, end_date=20240824T183658
[2024-08-24T18:36:58.934+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-24T18:36:58.965+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-24T18:36:58.967+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
